### PR TITLE
Add [string] contains [string] block

### PR DESCRIPTION
**Pull request by Kenny2github**
### Resolves

LLK/scratch-gui#601

### Proposed Changes

Adds a “contains” block to `scratch-blocks/core/blocks-vertical.js`

The block’s code is in LLK/scratch-vm#660

### CODE REVIEW

**Comment by thisandagain** [comment]
This is a good start! You will also need to modify the default toolbox here:
https://github.com/LLK/scratch-blocks/blob/develop/blocks_vertical/default_toolbox.js

**Comment by Kenny2github** [comment]
@thisandagain Done. Once Travis checks through this should be good to go.

File: blocks_vertical/default_toolbox.js [hunk 1]
```diff
@@ -580,6 +580,13 @@ Blockly.Blocks.defaultToolbox = '<xml id="toolbox-categories" style="display: no
         '</shadow>'+
       '</value>'+
     '</block>'+
+    '<block type="operator_contains" id="operator_contains">'+
+      '<value name="STRING1">'+
```
Comment (thisandagain) [inline blocks_vertical/default_toolbox.js]: You are missing a value definition for `STRING2`.

Comment (Kenny2github) [inline blocks_vertical/default_toolbox.js]: *facepalm*

**Comment by Kenny2github** [comment]
Apparently eslint on Travis crashes my iPad 😒 

@thisandagain Anyway, this should be (actually) done now - any final thoughts?

File: blocks_vertical/default_toolbox.js [hunk 2]
```diff
@@ -580,6 +580,18 @@ Blockly.Blocks.defaultToolbox = '<xml id="toolbox-categories" style="display: no
...
+        '<shadow type="text">'+
+          '<field name="TEXT">hello world</field>'+
```
Comment (thisandagain) [inline blocks_vertical/default_toolbox.js]: After looking at this in detail, I think we should switch this to `hello` and change the default for `STRING2` to `world` as this boolean reporter looks too long in the category menu.

**Comment by thisandagain** [comment]
Looks good! One minor comment and then I think this will be good to land.

**Comment by thisandagain** [comment]
Note to self: remember to squash these changes when merging.

File: blocks_vertical/default_toolbox.js (hunk 2 above)
Comment (Kenny2github) [reply blocks_vertical/default_toolbox.js]: So your intention is for it to report false by default? Sure.

**Comment by Kenny2github** [comment]
@thisandagain Done - this should be good once Travis finishes.

**Comment by Kenny2github** [comment]
@thisandagain any updates? /cc @ericrosenbaum

**Comment by Kenny2github** [comment]
@thisandagain all (1) check(s) passed - no conflicts - this should be ready to go, right?

**Comment by Kenny2github** [comment]
@thisandagain all checks passed, code looks good, this should be ready to go!

Status on this PR?

**Comment by thisandagain** [comment]
@Kenny2github Thanks for the follow-up, but please see the Milestone date before pinging on PRs and issues. This will be reviewed and landed prior to the Milestone date (in this case: September 20th).

**Comment by Kenny2github** [comment]
@thisandagain Ah, sorry - I was just anxious.

Though it’s strange knowing that the vm pull has been merged, while the block won’t actually appear in the palette until this is merged...

But thank you for clarifying.

**Review by thisandagain — APPROVED** [review]

**Comment by thisandagain** [comment]
Thanks @Kenny2github. This looks good.

**Comment by Kenny2github** [comment]
w00t! This is going to be so helpful!

**Pull request merged**